assets/js/components/KeyMetrics/ChipTabGroup/index.js

```diff
@@ -124,19 +229,30 @@ export default function ChipTabGroup( { allMetricItems, savedItemSlugs } ) {
 							) ) }
 						</div>
 					) ) }
-				{ isMobileBreakpoint &&
-					[ currentSelectionGroup, ...KEY_METRICS_GROUPS ].map(
-						( group ) => (
-							<Chip
-								key={ group.SLUG }
-								slug={ group.SLUG }
-								label={ group.LABEL }
-								isActive={ group.SLUG === isActive }
-								onClick={ onChipChange }
-								selectedCount={ selectedCounts[ group.SLUG ] }
-							/>
-						)
-					) }
+				{ isMobileBreakpoint && (
+					<TabBar
```

Comment:
To support proper horizontal scroll on mobile devices for tabs, existing `TabBar` component is used for simplicity, since it is already providing everything needed to make the scroll smooth and cross browser compatible, RTL supported, etc. No need to reinvent the wheel